### PR TITLE
Fixed #4 Live demo video link in README.md

### LINKED ISSUE #4: Live demo video link
The link seems to be broken for the video in docs_images/FlashType.avi

### PR DESCRIPTION
**Pull request by vikasganiga05**
## Fixes Issue

This PR fixes the following issues :
#4

## Changes proposed

The live demo video is fixed on the `README.md` file. I converted the `AVI` file into `GIF` which supported by github markdown syntax.
Thank You :pray: 

```diff
-![Live Demo](https://github.com/kaiwalyakoparkar/flashtype/blob/master/doc_images/FlashType.avi)
+![Live Demo](https://github.com/kaiwalyakoparkar/flashtype/blob/master/doc_images/FlashType.gif)
```

## Check List (Check all the boxes which are applicable)

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain copyrighted content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots 

screenshot-nimbus-capture-2021 03 24-01_35_40

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ This is a react based (side) project intended to practice help you in typing fas
 
 ## 📌Live Demo:
 
-![Live Demo](https://github.com/kaiwalyakoparkar/flashtype/blob/master/doc_images/FlashType.avi)
+![Live Demo](https://github.com/kaiwalyakoparkar/flashtype/blob/master/doc_images/FlashType.gif)
```
Comment (kaiwalyakoparkar) [inline README.md]: Change link to Live Demo link to (./doc_images/FlashType.gif) so that this can work on any fork

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

**Review by kaiwalyakoparkar — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (vikasganiga05) [reply README.md]: I fixed the link to gif

**Comment by vikasganiga05** [comment]
Thank You for accepting my pull request :pray:

**Comment by kaiwalyakoparkar** [comment]
Your efforts worth it! keep contributing to open source :+1:

**Pull request merged**